src/openrct2-ui/windows/TileInspector.cpp

```diff
@@ -1625,6 +1625,16 @@ static void window_tile_inspector_invalidate(rct_window* w)
             break;
         case TileInspectorPage::Wall:
         {
+            bool canBeSloped = false;
+            bool hasAnimation = false;
+            const auto sceneryEntry = tileElement->AsWall()->GetEntry();
```

Comment:
`tileElement->AsWall()` can return nullptr. Usually I don't mind if it checks the tile element type beforehand. But in this case it makes an assumption that the tile element is always the right type for the current page that is showing. So it it would be safer if we started checking nullptr after `AsWall` etc. in this method.

Reply:
Why don't we just change the `switch` condition to `tileElement->GetType()`? Then we can also be sure the type is correct and `AsWall() will not return `nullptr`.

Reply:
That is fine with me.